Update New York.md

aaaaaron-ship-it opened the pull request:
Added incident of protestor arrest by non-uniformed agents. Didn't say if they're police or federal because I'm not sure. I think this is something we cover but let me know if I'm wrong.

emarcey commented on reports/New York.md:
```diff
@@ -1498,6 +1498,21 @@ id: ny-newyorkcity-97
+
+A group of people not wearing any identification as law enforcement officers surround and grab a protestor. After a brief struggle, protestor is brought to the ground and then dragged into an unmarked van. Uniformed police show up on bicycles to prevent the other protestors from intervening and then the van drives away.
+
+tags: arrest, abuse-of-power
```
Please add the protestor tag

emarcey commented on reports/New York.md:
```diff
@@ -1498,6 +1498,21 @@ id: ny-newyorkcity-97
+
+* https://twitter.com/michellelhooq/status/1288253381883113475
+* https://www.reddit.com/r/2020PoliceBrutality/comments/hzrn57/wtf_nyc_just_a_little_while_ago/
+* https://twitter.com/Naddleez/status/1288232816451432453
```
Another link https://www.instagram.com/p/CDM7z3GJnxP/?igshid=1fi9w19fe6k5s

emarcey replied:
https://twitter.com/protest_nyc/status/1288354104507498497?s=20

aaaaaron-ship-it commented:
I added the protestor tag and the extra links.

It looks like we now have some more info on this - not sure if I should update the description. We know this was NYPD not feds or anything. Also we know she was arrested for graffiti and writing on police cameras.

I also saw a screenshot where the police technically had badges that were basically impossible for anyone to see. I don't know if I should edit the description to be something like "didn't identify themselves as officers" or something so we don't open this up to that criticism.

emarcey commented:
> not sure if I should update the description

Definitely update the description if we have more info. 

>  I don't know if I should edit the description to be something like "didn't identify themselves as officers" or something so we don't open this up to that criticism.

I would say change it to "didn't identify themselves as officers" and maybe add that badges were difficult to see

emarcey commented:
Another link: https://twitter.com/KyleLandrySax/status/1288303998395715585

emarcey commented:
Another new link: https://twitter.com/NYPDDetectives/status/1288493685571821568?s=19

aaaaaron-ship-it commented:
Added links, updated description. Also added link to news article. I decided just to not mention the badges - I don't think it's relevant as long as the original language "nothing identifying themselves as police" isn't there any more.

emarcey commented:
Thanks for being flexible on this one! It was a big incident that caught a lot of traction, so lots of things happening early on 👍
